packages/react-frontend/src/routes/AlbumInfo.jsx

```diff
@@ -126,7 +99,7 @@ export default function AlbumInfo() {
               {album.release_date.split("-")[0] + " "}
               {artist.artist_name} |{"  "}
               <span className="rating-stars">
-                {starRating}{" "}
+                <Rating name="album-rating" value={parseFloat(avgRating)} precision={0.5}/>
```

Comment:
It looks good, but when I hover over the average star rating, it changes to where my mouse is. Is it possible to make it read-only?

Reply:
oh wow, i didn't even notice it could hover like that haha, yes i think i should be able to change it to read-only and i will look into this a bit more. thanks for catching this, sara!

Reply:
also yes, i forgot to implement the precision in the individual reviews so ill change the individual reviews as well. I mainly focused on the one getting displayed in the title, but i agree this should be changed.